packages/react-instantsearch/package.json

```diff
@@ -30,6 +30,7 @@
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
     "react-native": "^0.41.2",
-    "react-test-renderer": "^15.5.4"
+    "react-test-renderer": "^15.5.4",
+    "react-navigation": "latest"
```

Comment:
"react-navigation": "latest" => will disappear we are not using it anymore :) Thanks for catching it.